ci/logging_tests_controller.yml

```diff
@@ -0,0 +1,184 @@
+#    # Elsewhere, we assume that we're already logged in
+#    - name: "Log into OCP"
+#      ansible.builtin.shell:
+#        cmd: |
+#          oc login -u kubeadmin -p "12345678" https://api.crc.testing:6443
+#      failed_when: false
+#      changed_when: false
+#
+#    # I don't think we need this, Zuul __should__ have taken care of this already
+#    - name: "Create id rsa file for ctlplane access"
+#      ansible.builtin.shell:
+#        cmd: |
+#          oc extract -n openstack secrets/dataplane-ansible-ssh-private-key-secret --to=- | grep -v "ssh\-rsa" > "{{ id_rsa_path }}"
+#      register: rsa_results
+#      failed_when: rsa_results.rc != 0
+#      changed_when: false
+#
+#    # (efoley): Once again, I don't think this is necessary; if it's not needed for zuul, because it's already done, but is needed before running this on other systems, then the README should be updated to reflect this;
+#    # A troubleshooting section can be added too so users can identify when they are missing these configuration steps
+#     - name: "Change files permissions"
+#       ansible.builtin.shell:
+#         cmd: |
+#           chmod 600 "{{ id_rsa_path }}"
+#       changed_when: false
+
+- name: "Verify logging projects, endpoints, credentials, nodes, pods, services, manifests and subscriptions"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOS0-12672"
+    common_pod_status_str: "Running"
+    common_pod_nspace: openstack-operators
+    common_pod_list:
+      - telemetry-operator-controller-manager
+      - dataplane-operator-controller-manager
+
+- name: "Verify logging pods are running in openstack"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12752"
+    common_pod_status_str: "Running"
+    common_pod_nspace: openstack
+    common_pod_list:
+      - openstackclient
+  tasks:
+    - name: "Verify Running Pods"
+      ansible.builtin.import_role:
+        name: common
+
+
+- name: "Verify logging pods are running in openshift-operators-redhat"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12673"
+    common_pod_status_str: "Running"
+    common_pod_nspace: openshift-operators-redhat
+    common_pod_list:
+      - loki-operator-controller-manager
+
+  tasks:
+    - name: "Verify Pods running"
+      ansible.builtin.import_role:
+        name: common
+        # vars can be passed here, so we can use the same play for this and the next play.
+      # vars:
+      #   - <vars from above>
+
+- name: "Verify logging pods are running in openshift-logging"
```

Comment:
These are passing except for ``logging-loki-gateway``, which has two instances, so the second task fails because there are ``\n`` in the pod name.